lib/line_drive/organization_fields.ex

```diff
@@ -0,0 +1,35 @@
+defmodule LineDrive.OrganizationFields do
+  @moduledoc """
+  This module encapsulates calls to the pipedrive organization fields resource API
+  """
+
+  use Tesla
+
+  alias LineDrive.OrganizationField
+  alias Tesla.Client
+
+  @callback get_org_field_keys_and_names(Client.t(), binary()) :: {:ok, OrganizationField.t()}
+
+  def get_org_field_keys_and_names(%Client{} = client, opts \\ []) do
+    start = Keyword.get(opts, :start, 0)
+    limit = Keyword.get(opts, :limit, 200)
+
+    client
+    |> get("/api/v1/organizationFields/", query: [start: start, limit: limit])
+    |> case do
+      {:ok, %Tesla.Env{status: 200, body: %{success: true, data: data}}} ->
+        org_fields =
+          data
+          |> Enum.map(fn item -> Map.take(item, [:key, :name]) end)
+          |> Enum.map(fn item -> OrganizationField.new(item) end)
```

Comment:
You could probably do this with a single Enum.map
```suggestion
          |> Enum.map(fn item ->
            item
            |> Map.take([:key, :name])
            |> OrganizationField.new()
          end)
```

Reply:
@alcwhite , thanks for the suggestion. I have made this change.